docs/preview/yamlgettingstarted-directory.md

```diff
@@ -0,0 +1,67 @@
+# YAML getting started - YAML directory (not available, for discussion only)
+
+## Summary
+
+We plan to support a convention-based directory structure, with a couple goals in mind:
+
+1. Today we only create a build definition on-push for the root file `.vsts-ci.yml`. And resource authorization on-push is only handled for that one file. With a convention based
```

Comment:
Does the file have to be in the root of the repo? If we put the file under another directory that isn't the root would it still work but just not auto update?

Reply:
root of the repo or the folder structure below if you want it to auto create.